src/helpers.js

```diff
@@ -250,7 +250,7 @@ export function checkDeprecations(opt, callback) {
   }
 
   if (opt.mangle) {
-    console.warn('marked(): mangle parameter is deprecated since version 5.0.0, should not be used and will be removed in the future. Instead use https://www.npmjs.com/package/marked-mangle.');
+    console.warn('marked(): mangle parameter is deprecated since version 5.0.0, should not be used and will be removed in the future. Note the "mangle" parameter is historically enabled by default, but can be manually disabled, or instead use https://www.npmjs.com/package/marked-mangle.');
```

Comment:
```suggestion
    console.warn('marked(): mangle parameter is deprecated since version 5.0.0, and will be removed in the future. Use `{mangle: false}` to hide this warning and optional enable using https://www.npmjs.com/package/marked-mangle.');
```